### DISCUSSION TITLE
ESP32-Rhasspy-Satellite support

### DISCUSSION THREAD

**Discussion by Omirax** [JarvyJ/HomeIntent#545] [General]
Hello! 
I like this project, I will start studying in the near future! 
Great job done!

Will satellites from this project be supported? Do I need a special configuration?
https://github.com/Romkabouter/ESP32-Rhasspy-Satellite

Regards,

**Comment by JarvyJ** [comment]
Hey,

Glad you're liking the project! Yes, it _should_ work with ESP32 satellites, I haven't tried an ESP32/microcontroller satellite, but as long as you pointed it at Home Intent's underlying Rhasspy instance - it should work.

On the Home Intent side all you have to do is add in the satellite id (follow the "Microcontroller/Other" setup on the [Satellites](https://homeintent.io/getting-started/advanced-features/satellites/#setting-up-the-satellite) page).

Good luck and let me know if it works!
